test/get/git/path_test.dart

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+import 'package:pub/src/io.dart';
+
+import '../../descriptor.dart' as d;
+import '../../test_pub.dart';
+
+main() {
```

Comment:
This is totally fine by default (and is already doable with, say a `path` dependency). I'm not sure it's worth adding a separate test, though, since it doesn't tickle any different code paths.